themes/hugo-rladies/layouts/partials/author.html

```diff
@@ -0,0 +1,16 @@
+
+<ul class="list-group">
+    {{ range . }}
+        <li class="list-group-item border-0">
+            {{ with .url }}
```

Comment:
where does the author's data come from?

Reply:
oh I see. should we plan to create Markdown files under `author/` instead?

Reply:
We likely should start preparing something like that. I'd ideally like to use the directory as author information (and also information on global team members etc). Like, the directory being our source of information on individuals?